src/components/Notification/Stories/BasicNotifDrawerStory.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { boolean } from '@storybook/addon-knobs';
+import { inlineTemplate } from '../../../../storybook/decorators/storyTemplates';
+import { DOCUMENTATION_URL } from '../../../../storybook/constants';
+import { Notification } from '../index';
+import { NotifDrawer } from '../NotifDrawer/index';
+import { Icon } from '../../Icon';
+import { MenuItem } from '../../MenuItem';
+import { Button } from '../../Button';
+
+const handleClick = e => {
+  e.preventDefault();
+  action('some on click')();
+};
+const expandedBool = boolean('Expanded Drawer', false);
+const hideBool = false;
+const basicNotifDrawerStory = stories => {
+  stories.addWithInfo('Notification Drawer', '', () => {
+    const story = (
+      <nav className="navbar navbar-pf-vertical">
+        <nav className="collapse navbar-collapse">
+          <ul className="nav navbar-nav navbar-right navbar-iconic">
+            <li className="drawer-pf-trigger open">
+              <a href="#0" className="nav-item-iconic" onClick={handleClick}>
```

Comment:
I'm wondering...  would href="#" bring yah to the top of page given line 13?  Is `#0` necessary here?

Reply:
Hi Allen, i copied the Navbar html becuase there is no react implementation, `handleClick()` prevents the default function so clicking on the bell in this specific story won't do a thing.
anyway i can remove the `href`

Reply:
I guess the real question is... does the href of an a register as an event interface... I know mdn hates re-purposing anchors as buttons... https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Notes ...